event_model/tests/test_em.py

```diff
@@ -121,7 +135,7 @@ def test_round_trip_pagination():
     assert actual == expected
 
 
-def test_bulk_events_to_event_page():
+def test_bulk_events_to_event_page(tmp_path):
```

Comment:
@gwbischof, @danielballan, does it need to be imported somewhere? Or pytest will automatically understand it should come from fixtures?

Reply:
I think pytest assumes that all arguments for functions that begin with "test" are fixtures.

Reply:
I see 👍. It is also used the same way in the examples from @danielballan's link in https://github.com/NSLS-II/event-model/pull/59#issuecomment-476302108.